Apps/WebClient/src/ClientApp/src/shims-vue.d.ts

```diff
@@ -24,3 +24,4 @@ declare module "vue-loading-overlay";
 declare module "vue-fullcalendar";
 declare module "vue-content-placeholders";
 declare module "vue-clickaway";
+declare module "@types/winston;"
```

Comment:
I don't think we need this one. Also for this PR, we don't need to add this line right?

Reply:
The issue is we will have build errors if we don't declare a type file for the package. Leaving in unless there is an issue with it being there

Reply:
If you don't have any build errors when building this locally, then the Build agent shouldn't have any errors. When introducing this library, we didn't encounter any errors. If this line helps to resolve your local build then we can add this, if not, let's remove it then?

Reply:
removed